src/mantarray_waveform_analysis/compression_cy.pyx

```diff
@@ -34,9 +34,7 @@ def rsquared(int [:] x_values, int [:] y_values):
     intercept = -slope * x_1 + y_1
 
     # based on https://stackoverflow.com/questions/893657/how-do-i-calculate-r-squared-using-python-and-numpy
-    ss_res = 0
-    ss_tot = 0
```

Comment:
do floats/ints automatically initialize to 0 in Cython?

Reply:
I'm not actually sure, but I noticed that those two variables were actually initialized to 0 a few lines down in the code so I thought it would be redundant to do it twice

Reply:
ah, so they are. I missed that